cluster/examples/kubernetes/ceph/admission-controller/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# deploy.sh
+# Sets up the environment for the admission controller webhook in the active cluster.
+set -eo pipefail
+
+# Set our known directories and parameters.
+BASE_DIR=$(cd "$(dirname "$0")"/..; pwd)
```

Comment:
Why is `BASE_DIR` the parent directory? If it were the current directory of the test instead of the parent, this script wouldn't need to be different from the copy under `tests/scripts`, right?

Reply:
It makes sense, i've changed the BASE_DIR to point to current dir